metals/src/main/scala/scala/meta/internal/metals/codeactions/ActionableDiagnostic.scala

```diff
@@ -0,0 +1,61 @@
+package scala.meta.internal.metals.codeactions
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals._
+import scala.meta.pc.CancelToken
+
+import org.eclipse.{lsp4j => l}
+
+class ActionableDiagnostic() extends CodeAction {
+  override def kind: String = l.CodeActionKind.QuickFix
+
+  override def contribute(
+      params: l.CodeActionParams,
+      token: CancelToken,
+  )(implicit ec: ExecutionContext): Future[Seq[l.CodeAction]] = {
+
+    def path =
+      params.getTextDocument.getUri.toAbsolutePath.toURI.toString
```

Comment:
is `params.getTextDocument.getUri.toAbsolutePath.toURI.toString` the same as `params.getTextDocument.getUri`?

Reply:
Yes, it should be the same. I fixed it.